server.js

```diff
@@ -4960,6 +4960,27 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Facebook integration.
+camp.route(/^\/facebook\/share\/([^\/]+)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var scheme = match[1]; // eg, https
+  var path = match[2];   // eg, shields.io
+  var format = match[3];
+  var page = encodeURIComponent(scheme + '://' + path);
+  var badgeData = getBadgeData('Share', data);
```

Comment:
Could you use a lowercase `'share'` please? We usually use a lowercase version.

(Note that the social badges get capitalized elsewhere.)